src/gluonts/model/npts/_model.py

```diff
@@ -168,12 +168,11 @@ def predict(
         # of the prediction range, and the frequency of the time series.
         samples_pred_range = samples[:, train_length:]  # prediction range only
 
+        freq = targets.index.freq.freqstr
         forecast_start = targets.index[-1] + 1 * targets.index.freq
 
         return SampleForecast(
-            samples=samples_pred_range,
-            start_date=forecast_start,
-            freq=forecast_start.freqstr,
+            samples=samples_pred_range, start_date=forecast_start, freq=freq
         )
```

Comment:
I don't understand why this change is necessary.

Reply:
In pandas versions older than 0.25 the frequency is not carried along when the time stamp is shifted.